py_zipkin/zipkin.py

```diff
@@ -182,7 +190,25 @@ def __init__(
         self.binary_annotations = binary_annotations or {}
         self.port = port
         self.sample_rate = sample_rate
-        self.include = include
+        self.kind = None
+        # If `kind` is not set, then we generate it from `include`.
+        # This code maintains backward compatibility with old versions of py_zipkin
+        # which used include rather than kind to identify client / server spans.
+        if kind is None and include:
+            # If `include` contains only one of `client` or `server`
+            # than it's a client or server span respectively.
+            # If neither or both are present, then it's a local span
+            # which is represented by kind = None.
+            if 'client' in include and 'server' not in include:
+                self.kind = Kind.CLIENT
+            elif 'client' not in include and 'server' in include:
+                self.kind = Kind.SERVER
+            else:
+                self.kind = Kind.LOCAL
+            warnings.warn(
+                'The include argument is deprecated. Please use kind.',
+                DeprecationWarning,
```

Comment:
I know I suggested this and I definitely wanna warn the user, but i wonder if this is going to spam the logs too much (and if that's ok).

I wish we had compile-time warnings...